website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -90,6 +103,14 @@ operator-sdk create api \
     --controller
 ```
 
+### How to keep `apiextensions.k8s.io/v1beta1` for CRDs?
+
+From now on, the CRDs that will be created by controller-gen will be using the Kubernetes API version `apiextensions.k8s.io/v1`  by default, instead of `apiextensions.k8s.io/v1beta1`. 
```

Comment:
```suggestion
From now on, the CRDs that will be created by controller-gen will be using Kubernetes API version `apiextensions.k8s.io/v1` by default, instead of `apiextensions.k8s.io/v1beta1`. 
```